### PR TITLE
Add leaderboard basic structure

### PR DESCRIPTION
**Pull request by bimbolabuari**
# Leaderboard Microverse Project 🏅 

 > This PR adds a project about  Webpack that showcases the knowledge of HTML, CSS, Webpack Configuration, Javascript, import and export modules.

## Built With  🙂 

- Semantic HTML & CSS
- Linters (Stylelint, Web hint, Lighthouse)

## Live Preview 🚀 

[Live Preview Link](https://deploy-preview-1--bimbola-mv-leaderboard.netlify.app/)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
Comment (mimipeshy) [inline .gitignore]: - [ ] Kindly remove the dist folder from .gitignore and have it as part of your project structure.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <section class="form-section">
+       <h2 class="form-section-header">Add your score</h2>
+       <form>
+           <label><input type="text" placeholder="Your name"></label><br>
+           <label><input type="number" placeholder="Your score"></label><br>
+      <button type="submit">Submit</button>
+        </form>
+    </section>
+</main>
+<footer class="footer"></footer>
```
Comment (mimipeshy) [inline src/index.html]: - [ ] Kindly move all this code into `index.js` file. The goal is to get you comfortable with working with proper ES6 styling such as using `arrow functions`, proper usage of `const`, and `let` keywords.
- [ ] The index.html body should contain only the root element which is a link to the index.js file

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @bimbolabuari  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights
- Good GitHub workflow ✔️ 
- Not forming any linter errors ✔️
- You get extra credits for hosting you website ✔️ 👏🏾 👏🏾  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by dasileker — APPROVED** [review]
### **Hello @bimbolabuari** 

### Project Approved 💯💯✅

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 

### **Additional Information**

- 

- For the JavaScript module please use the 👉 [Javascript Linters](https://github.com/microverseinc/linters-config/tree/master/javascript), not the `**html/css/js**` linters.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**